src/marking.js

```diff
@@ -1,30 +1,37 @@
-/** @module marking */
+/** 
+ * @module marking 
+ * @license MIT
+ */
 
 /**
- * Given a set of raw per-element score statistics, determine which contribute most to total posterior probability,
- * and are thus members of the credible set.
- * @param {Number[]} statistics Calculated statistics used to rank the credible set (eg raw bayes factors)
- * @param {Number} [cutoff=0.95] Keep taking items until we have accounted for >= this fraction of the total probability
- * @return {Number[]} An array with posterior probabilities (for the items in the credible set), and zero for all
- *   other elements
- *  This array should be the same length as the provided statistic array
+ * Given an array of probabilities, determine which elements of the array fall within the X% credible set, 
+ * where X is the cutoff value. 
+ * 
+ * Return a mask of `probs`, where values for elements that do not belong to the credible set are set to 0. 
```

Comment:
Thanks for the documentation updates- although I made some small updates yesterday that conflict, overall this is a big improvement for the language and tone as we gain insight into this method.

I notice that you may have been editing a slightly older version of the branch- I may make some changes after merging to sync up cases where last night's revisions changed the meaning/explanation of the function.

(eg this function doesn't return a mask; it converts to posterior probabilities)

In this module, the assumption is that you could pass in any score, not just bayes factors or posterior probabilities. Is there a reasonable use case (eg different credible set method) where the word "probs" would be too restrictive?

Reply:
In functions that calculate credible sets, the only thing that really makes sense is passing in a probability (by the definition of a credible set.) 

Actually I think I missed that in one other location (will go back and check.)